content/docs/intro/vs/serverless.md

```diff
@@ -30,3 +29,5 @@ the Serverless Framework](
 https://serverless.com/blog/serverless-application-for-long-running-process-fargate-lambda/) takes about 38 pages
 of explanation, including manual steps and AWS CloudFormation; [the after code using Pulumi](
 https://github.com/pulumi/examples/tree/master/cloud-js-thumbnailer) is only about 38 lines of code.
+
+Follow our blog for updates on [new features] ({{< relref "/blog/tag/new-features" >}}), and if you have any questions about Pulumi, feel free to [contact us] ({{< relref "/contact" >}}) any time.
```

Comment:
This paragraph seems bolted-on and doesn't seem to be adding much keyword-relevant info, so suggest simply removing it.
```suggestion
```